pkg/kubelet/kubelet.go

```diff
@@ -1326,8 +1316,23 @@ func (kl *Kubelet) initializeRuntimeDependentModules() {
 		// TODO(random-liu): Add backoff logic in the babysitter
 		glog.Fatalf("Failed to start cAdvisor %v", err)
 	}
+	// trigger on-demand stats collection once so that we have capacity information for ephemeral storage.
+	// ignore any errors, since if stats collection is not successful, the container manager will fail to start below.
```

Comment:
you mean kl.containerManager.Start will fail? What is the reason?

Reply:
It will fail because this PR makes it return an error if cadvisor.RootFsInfo returns an error

Reply:
got it. Just in case, it takes a little bit long (minutes) to start cadvisor, will it be ok?

Reply:
I'm not sure why it would take minutes to start.  The reason we had to wait previously is that we had to wait for periodic stats collection to run once.  We are forcing that to happen immediately here, so we dont have a delay anymore.

Reply:
Now container manager is started a little bit later (in updateRuntimeUp) than before (in initializeModules), just want to confirm there is no issue there. Otherwise, lgtm.